kubernetes/structures_container.go

```diff
@@ -1,7 +1,7 @@
 package kubernetes
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This line is equivalent to one it replaces. The convention is to rename imports when there is a conflict between multiple import paths with the same last element. In this case we only import one package path ending in v1, so there is no need to use the renamed import form.

Reply:
Oops, yeah this was my editor.